docs/source/api.md

```diff
@@ -0,0 +1,53 @@
+# Getting started with the HTTP API
```

Comment:
Maybe name this section http-api-tutorial.md? api.md is ambiguous as there is an HTTP API, and each driver will also have its own API (e.g. the Python API provided by the Python driver, the Java API provided by the Java driver, etc.). If you do change the name, remember to update it in `index.rst` as well.